cmd/utils.go

```diff
@@ -83,18 +83,21 @@ func checkWatchesConfiguration() {
 		return
 	}
 
-	l := strings.TrimSuffix(string(f), "\n")
-	c, err := strconv.Atoi(l)
-	if err != nil {
-		log.Infof("Fail to parse the value of  max_user_watches: %s", err)
-		return
-	}
-
-	log.Debugf("max_user_watches = %d", c)
-
-	if c <= 8192 {
-		log.Yellow("The value of /proc/sys/fs/inotify/max_user_watches is too low. This can affect Okteto's file synchronization performance.")
+	if isWatchesTooLow(string(f)) {
+		log.Yellow("The value of /proc/sys/fs/inotify/max_user_watches is too low.")
+		log.Yellow("This can affect Okteto's file synchronization performance.")
 		log.Yellow("We recommend you to raise it to at least 524288 to ensure proper performance.")
```

Comment:
we are checking for 8192 in the function but on the message we recommend 524288. Is that expected?